grails-app/services/org/broadinstitute/orsp/ConsentCollectionLinkService.groovy

```diff
@@ -0,0 +1,16 @@
+package org.broadinstitute.orsp
+
+import groovy.util.logging.Slf4j
+
+@Slf4j
+class ConsentCollectionLinkService {
+
+    ConsentCollectionLink save(ConsentCollectionLink consentCollectionLink) {
+        consentCollectionLink.save(flush: true)
+    }
+
+    void delete(ConsentCollectionLink consentCollectionLink) {
+        consentCollectionLink?.delete(flush: true)
+    }
+}
+
```

Comment:
We already have a class for this, [PersistenceService](https://github.com/broadinstitute/orsp-pub/blob/feature/grails-app/services/org/broadinstitute/orsp/PersistenceService.groovy)